src/SensorStation.ino

```diff
@@ -0,0 +1,293 @@
+/*
+This script represents a sensor station with three sensors attached:
+  * DHT22 for humidity and temperature,
+  * TSL2561 for luminosity,
+  * CCS811 for eCO2 and TVOC
+
+The station reads the sensor data in regular interval and publishes them to MQTT topics.
+
+Run this script on a ESP8266 board.
+*/
+
+#include <Wire.h>
+#include "ccs811.h"
+#include "DHT.h"
+#include <Adafruit_Sensor.h>
+#include <Adafruit_TSL2561_U.h>
+#define ARDUINOJSON_USE_LONG_LONG 1
+#include <ArduinoJson.h>
+#include <NTPClient.h>
+#include <tuple>
+#include <WiFiUdp.h>
+#include <string>
+using namespace std;
+#include <ESP8266WiFi.h>
+#include <PubSubClient.h>
+#include "wifi_secrets.h"
+#include "mqtt_secrets.h"
+#include "sensor_station.h"
+
+const long utcOffsetInSeconds = 3600;
+
+std::string MQTT_CLIENT_PREFIX = "sensor_station_";
+std::string STATION_ID_STR = STATION_ID;
+std::string MQTT_CLIENT_ID_STR = MQTT_CLIENT_PREFIX + STATION_ID_STR;
+char *MQTT_CLIENT_ID = &MQTT_CLIENT_ID_STR[0];
+
+CCS811 ccs(CCS811_NWAKE_PIN, CCS811_SLAVE_ADDR);
+DHT dht(DHT22_DATA_PIN, DHT_TYPE);
+Adafruit_TSL2561_Unified tsl = Adafruit_TSL2561_Unified(TSL2561_ADDR, TSL2561_SENSOR_ID);
+
+WiFiClient espClient;
+PubSubClient client(espClient);
+
+WiFiUDP ntpUDP;
+NTPClient timeClient(ntpUDP, "pool.ntp.org", utcOffsetInSeconds);
+
+void setup_wifi()
+{
+  WiFi.begin(WIFI_SSID, WIFI_PSK);
+  while (WiFi.status() != WL_CONNECTED)
+  {
+    delay(100);
+  }
+  Serial.println(WiFi.localIP());
+}
+
+void setup_mqtt()
+{
+  client.setServer(MQTT_BROKER, 1883);
+}
+
+void setup_ccs811()
+{
+  ccs.set_i2cdelay(50); // Needed for ESP8266 because it doesn't handle I2C clock stretch correctly
+  if (!ccs.begin())
+  {
+    Serial.println("CSS811 begin FAILED");
+    while (1)
+      ;
```

Comment:
You can put this after while (1), no need to have this semicolon on its own line